vision/google/cloud/vision_helpers/decorators.py

```diff
@@ -96,12 +99,14 @@ def _create_single_feature_method(feature, enum):
     feature_value = {'type': enum.__dict__[feature]}
 
     # Define the function to be returned.
-    def inner(self, image, retry=None, timeout=None, **kwargs):
+    def inner(self, image, max_results=2**31-1,
+              retry=None, timeout=None, **kwargs):
```

Comment:
It would be better to use `None` as the sentinel value, and just leave it out of the `feature_value` mapping below.